### PR TITLE
Added placeholder for Split's textEdit

### PR DESCRIPTION
**Pull request by zneix**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description
Added a placeholder for textEdit split which currently looks like this:

https://cdn.zneix.eu/FmE1NIn.png
farming commits xdddddddddd

### CODE REVIEW

**Comment by zneix** [comment]
Just had a random thoguht - since we already connect to a account change signal, we might as well try connecting to other events if needed and try implementing something like request from #870

File: src/widgets/splits/Split.hpp [hunk 1]
```diff
@@ -118,6 +120,7 @@ class Split : public BaseWidget, pajlada::Signals::SignalHolder
     pajlada::Signals::Connection roomModeChangedConnection_;
 
     pajlada::Signals::Connection indirectChannelChangedConnection_;
+    pajlada::Signals::SignalHolder SignalHolder_;
```
Comment (pajlada) [inline src/widgets/splits/Split.hpp:123]: should be `signalHolder_` not `SignalHolder_`

**Review by pajlada — CHANGES_REQUESTED** [review]

File: src/widgets/splits/Split.hpp (hunk 1 above)
Comment (zneix) [reply src/widgets/splits/Split.hpp]: Fixed in 7ee5716b

Comment (zneix) [reply src/widgets/splits/Split.hpp]: Fixed in 7ee5716b

**Review by pajlada — APPROVED** [review]

**Pull request merged**